src/main/java/org/symphonyoss/integration/jira/authorization/JiraAuthorizationManager.java

```diff
@@ -305,11 +300,13 @@ public boolean isUserAuthorized(IntegrationSettings settings, String url, Long u
       HttpResponse response =
           provider.makeAuthorizedRequest(jiraOAuth1Data.getAccessToken(), myselfUrl,
               HttpMethods.GET, null);
-
       return response.getStatusCode() != HttpStatusCodes.STATUS_CODE_UNAUTHORIZED;
     } catch (MalformedURLException e) {
       throw new JiraOAuth1Exception(logMessage.getMessage("integration.jira.url.api.invalid", url),
           e, logMessage.getMessage("integration.jira.url.api.invalid.solution"));
+    } catch (OAuth1HttpRequestException e) {
+      throw new JiraOAuth1Exception(logMessage.getMessage("integration.jira.url.api.invalid", url),
+          e, logMessage.getMessage("integration.jira.url.api.invalid.solution"));
```

Comment:
Change this message to: integration.jira.unreachable

integration.jira.unreachable = "Cannot access JIRA instance"
integration.jira.unreachable.solution = "Make sure the JIRA instance {0} is working fine"